components/JsonEditor.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef, MutableRefObject } from 'react'
+import BaseJsonEditor from 'jsoneditor'
+import 'jsoneditor/dist/jsoneditor.css'
+
+export const JsonEditor = ({
```

Comment:
i think @natlotzy had a good point. can we use default exports for components like airbnb https://github.com/airbnb/javascript/tree/master/react#naming

(i.e. i think we can remove this export)

Reply:
Done